modules/understanding-upgrade-channels.adoc

```diff
@@ -94,13 +94,13 @@ endif::openshift-origin[]
 The service recommends only upgrades that have been tested and have no serious issues. It will not suggest updating to a version of {product-title} that contains known vulnerabilities. For example, if your cluster is on {product-version}.1 and {product-title} suggests {product-version}.4, then it is safe for you to update from {product-version}.1 to {product-version}.4. Do not rely on consecutive patch numbers. In this example, {product-version}.2 is not and never was available in the channel.
 
 ifndef::openshift-origin[]
-Update stability depends on your channel. The presence of an update recommendation in the `candidate-{product-version}` channel does not imply that the update is supported. It means that no serious issues have been found with the update yet, but there might not be significant traffic through the update to suggest stability. The presence of an update recommendation in the `fast-{product-version}` or `stable-{product-version}` channels at any point is a declaration that the update is supported. While releases will never be removed from a channel, update recommendations that exhibit serious issues will be removed or made conditional from all channels. When an update recommendation is supported, it remains supported for the life of {product-version}, even if the update recommendation is later dropped or made conditional.
+Update stability depends on your channel. The presence of an update recommendation in the `candidate-{product-version}` channel does not imply that the update is supported. It means that no serious issues have been found with the update yet, but there might not be significant traffic through the update to suggest stability. The presence of a release in `fast-{product-version}`, `stable-{product-version}`, or `eus-4.y` channels is a declaration that the release is supported. While releases will never be removed from a channel, update recommendations that exhibit serious issues will be removed or made conditional from all channels. All updates between supported releases are supported, regardless of whether they are or have ever been recommended.
 
-Red Hat will eventually provide supported update paths from any supported release in the `fast-{product-version}` or `stable-{product-version}` channels to the latest release in {product-version}.z, although there can be delays while safe paths away from troubled releases are constructed and verified.
+Red Hat will eventually provide recommended update paths from any supported release to the latest supported release in {product-version}.z in the `fast-{product-version}`, `stable-{product-version}`, and `eus-4.y` channels, although there can be delays while safe paths away from troubled releases are constructed and verified. The recommended update path to reach the latest {product-version}.z may require multiple updates.
```

Comment:
> The recommended update path to reach the latest {product-version}.z may require multiple updates.

When and why would that be the case? We're committed to z-stream updates being cumulative and no need to jump through intermediate z-streams.